lib/generators/spotlight/assets/propshaft_generator.rb

```diff
@@ -45,7 +45,14 @@ def add_bootstrap
       def install_gems
         gem 'jsbundling-rails'
         gem 'cssbundling-rails'
-        run 'bundle install'
+      end
+
+      def bundle_install
```

Comment:
Is this required?  Generators should always be run in the context of the app.

Reply:
Sort of. I've been searching for a better solution for this. We were loosely copying https://github.com/geoblacklight/geoblacklight/blob/36fa3977309891a026baf43516bff7bc7f4a6803/lib/generators/geoblacklight/assets/vite_generator.rb#L26

The issue is that we're adding cssbundling-rails/jsbundling-rails in the generator. They might not have them installed and I've been on the fence if it's correct to add them to the gemspec, because they aren't necessarily dependencies in a non-propshaft configuration.